### ISSUE TITLE
Learn and playground are not blocked on mobile

### ISSUE THREAD

**Issue by ShaiBer** [winglang/wing#3583]
### I tried this:

https://www.winglang.io/learn/ and https://www.winglang.io/play/ on mobile

### This happened:

I was allowed in

### I expected this:

Get a message to use a computer

### Component

Playground, Tutorial

### Community Notes

* Please vote by adding a 👍 reaction to the issue to help us prioritize.
* If you are interested to work on this issue, please leave a comment.

**Comment by skyrpex** [comment]
What's the motivation behind this? That mobiles are too small? If that's the motivation, we could show this message based on the device width.

**Comment by ekeren** [comment]
> What's the motivation behind this? That mobiles are too small? If that's the motivation, we could show this message based on the device width.

Right, width is a good indicator of a bad experience in learn.

**Comment by skorfmann** [comment]
I agree that the https://www.winglang.io/learn/ view doesn't really work well - even on a large iphone. However, the https://www.winglang.io/playground is not too bad actually. Also, I believe these two sites could be serving different purposes:

1. learn: your really want to interact and use the website, that's the entire point of this
2. playground: Likely that one just want to checkout a link posted in Slack / Github issues / etc - for that, it's definitely good enough
![IMG_2321](https://github.com/winglang/wing/assets/136789/f022909d-391e-4042-b2af-fb4bc34e0022)

**Comment by ShaiBer** [comment]
Can you edit code on the iPhone? for some reason it doesn't really work on
my Android (but it might be my specific vendor)

**Comment by skorfmann** [comment]
> Can you edit code on the iPhone?

yes, I can edit. It's fully working with snappy ux

**Comment by ShaiBer** [comment]
Nice!
